src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Circle extends FigureForCircle {
+    public Circle(String color, int radius) {
+        super(color, radius);
+    }
```

Comment:
rewrite code like so in other figures to avoid redundant `FigureFor...` classes
```suggestion
    private final int radius;

    public Circle(String color, int radius) {
        super(color);
        this.radius = radius;
    }
```